src/components/NetflowTraffic.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  useResolvedExtensions,
+  isModelFeatureFlag,
+  ModelFeatureFlag,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+const NetflowTraffic: React.FC = () => {
+  const [extensions] = useResolvedExtensions<ModelFeatureFlag>(isModelFeatureFlag);
+
+  return !_.isEmpty(extensions) ? (
+    <div>
```

Comment:
is there anything special that I should do to see some content here?
I've changed that content locally with some dummy text (`<div>TEST</div>`) but it doesn't show up on the console, on the `/netflow-traffic` page, it's still a blank page

Reply:
I stopped my test when I had this blank page and the sub-section in the left panel.

Let met test it.